src/viadot/orchestration/prefect/utils.py

```diff
@@ -19,6 +19,8 @@
 
 from prefect.client.orchestration import PrefectClient
 from prefect.settings import PREFECT_API_KEY, PREFECT_API_URL
+from prefect_sqlalchemy import DatabaseCredentials
```

Comment:
It looks like an external package, please add it to `pyproject.toml` if it's needed